CDF Project Social Media Guide.md

```diff
@@ -0,0 +1,123 @@
+# Social Media Guide for CDF Projects
+
+## CD Foundation Social Accounts
+
+If you want the CD Foundation to repost what you post, tag us on social media:
+
+* **Twitter:**[@CDeliveryFdn](https://twitter.com/CDeliveryFdn)
+* **LinkedIn:** [@Continuous Delivery Foundation](https://www.linkedin.com/company/cdeliveryfdn)
+* **YouTube:** [@CDeliveryFdn](https://www.youtube.com/@cdeliveryfdn)
+
+
+## Official Hashtags
+Here are the hashtags we use for conferences and general tweeting.
+
+
+### Conferences and events
+* #cdCon
+* #CDSummit
+* #CDMiniSummit
+
+
+### General
+* #ContinuousDelivery
+* #DevOps
+* #GitOps
+
+
+## Project Accounts
+
+### Twitter
```

Comment:
Maybe this could be X (Twitter) to reflect the new name

Reply:
We can do that but I was resisting the silliness of the new name :D